components/Onboarding/Sections/Profile.js

```diff
@@ -0,0 +1,193 @@
+    height: PORTRAIT_SIZE_S,
+    [mediaQueries.mUp]: {
+      width: PORTRAIT_SIZE_M,
+      height: PORTRAIT_SIZE_M
+    }
+  }),
+  actions: css({
+    marginBottom: 20,
+    display: 'flex',
+    flexWrap: 'wrap',
+    position: 'relative',
+    '& > button': {
+      flexGrow: 1,
+      margin: '5px 15px 0 0',
+      minWidth: '120px',
+      [mediaQueries.mUp]: {
+        flexGrow: 0,
+        margin: '5px 15px 0 0',
+        minWidth: '160px'
+      }
+    }
+  })
+}
+
+const CONDENSED_PROFILE = gql`
+  query condensedProfile {
+    me {
+      id
+      firstName
+      lastName
+      portrait
+      statement
+      username
+      isAdminUnlisted
+      isListed
+      isEligibleForProfile
+      hasPublicProfile
+    }
+  }
+`
+
+class Profile extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      values: {},
+      errors: {},
+      dirty: {}
+    }
+
+    this.onChange = fields => {
+      this.setState(FieldSet.utils.mergeFields(fields))
+    }
+
+    this.onCompleted = () => {
+      props.onContinue.apply(null, [props])
+    }
+  }
+
+  render () {
+    const { onContinue } = this.props
+    const { values: stateValues, errors, dirty } = this.state
+
+    return (
+      <Query query={CONDENSED_PROFILE}>
+        {({ loading, error, data: { me: queryValues } = {} }) => {
+          const mergedValues = Object.assign({}, queryValues, stateValues)
+          const user = queryValues
+
+          return (
+            <Section
+              heading='Profile'
+              showContinue={!!error}
+              isLoading={loading}
+              error={error}
+              isTicked={user && user.hasPublicProfile}
+              {...this.props}>
+              <P {...styles.p}>Das Profil ist Ihre Visitenkarte bei der Republik.</P>
+              <P {...styles.p}>Wir verlinken Ihr Profil zum Beispiel bei Ihren Dialogbeiträgen.</P>
+
+              <div {...merge(styles.portrait)}>
+                <Portrait
+                  user={user}
+                  isEditing
+                  isMe
+                  values={mergedValues}
+                  errors={errors}
+                  dirty={dirty}
+                  onChange={this.onChange} />
+              </div>
+              <div {...styles.field}>
+                <Statement
+                  user={user}
+                  isEditing
+                  isMe
+                  onChange={this.onChange}
+                  values={mergedValues}
+                  errors={errors}
+                  dirty={dirty} />
+              </div>
+              <div {...styles.field}>
+                <UsernameField
+                  user={user}
+                  isEditing
+                  isMe
+                  onChange={this.onChange}
+                  values={mergedValues}
+                  errors={errors}
+                  dirty={dirty} />
+              </div>
+              <div {...styles.checkbox}>
+                <ListedCheckbox
```

Comment:
- [ ] Hint to user that community listed is only possible if portrait and statement is available